src/orders/orders.service.ts

```diff
@@ -21,6 +23,8 @@ export class OrdersService {
     private readonly orderModel: ReturnModelType<typeof OrdersSchema>,
     @Inject(forwardRef(() => SessionService))
     private readonly sessionService: SessionService,
+    @Inject(AddonsService)
```

Comment:
Unnecesary,please Remove it.

Reply:
but I use this in my recent function (at line 131) `await this.addonsService.disableAddons(disableAddons);`

Reply:
Sorry I am understanding now.